text/0984-drop-safari-less-than-14.md

```diff
@@ -0,0 +1,318 @@
+        <ul>
+          <li>Google Chrome</li>
+          <li>Mozilla Firefox</li>
+        </ul>
+      </EsCard>
+    </ul>
+
+-    <h3 class="text-center">Non-evergreen</h3>
+-
+-    <div class="layout">
+-      <ul class="list-unstyled layout lg:col-4 lg:start-2 my-3">
+-        <EsCard class="lg:col-3">
+-          <div class="text-center text-md">Desktop</div>
+-          <ul>
+-            <li>Safari</li>
+-          </ul>
+-        </EsCard>
+-        <EsCard class="lg:col-3">
+-          <div class="text-center text-md">Mobile</div>
+-          <ul>
+-            <li>Safari</li>
+-          </ul>
+-        </EsCard>
+-      </ul>
+-    </div>
+
+    <p>
+      For evergreen browsers, the minimum version of the browser that we support is
+      determined at the time of every minor release, following this formula:
+    </p>
+
+    <div class="layout my-3">
+      <div class="card">
+        <div class="card__content">
+          <p>Whichever browser version is greater/more recent out of:</p>
+
+          <ol>
+            <li>
+              The lowest/least recent version that fulfills any one of these properties:
+              <ul>
+                <li>It is the latest version of the browser.</li>
+                <li>It is the latest LTS/extended support version of the browser (such as Firefox ESR).</li>
+                <li>It has at least <em>0.25%</em> of global marketshare usage across mobile and
+              desktop, based on <a href="https://gs.statcounter.com/">statcounter</a>.</li>
+              </ul>
+            </li>
+            <li>
+              The minimum version supported in the previous release
+            </li>
+          </ol>
+        </div>
+      </div>
+    </div>
+
+    <p>
+      To simplify, the supported version either moves forward or stays the same for
+      each release based on overall usage and LTS/current release versions.
+    </p>
+
+    <p>
+      For non-evergreen browsers, support is locked at a specific major version, and
+      we support all major versions above that version.
++     However, all supported browsers are considered ever green.
+    </p>
+
+-    <div class="layout">
+-      <ul class="list-unstyled layout lg:col-4 lg:start-2 my-3">
+-        <EsCard class="lg:col-3">
+-          <div class="text-center text-md">Desktop</div>
+-          <ul>
+-            <li>Safari: 12</li>
+-          </ul>
+-        </EsCard>
+-        <EsCard class="lg:col-3">
+-          <div class="text-center text-md">Mobile</div>
+-          <ul>
+-            <li>Safari: 12</li>
+-          </ul>
+-        </EsCard>
+-      </ul>
+-    </div>
+
+    Within a version of a browser, we only support the most recent patch release.
+  </section>
+</div>
+```
+
+## Drawbacks
+
+More calculations for us to do when determining browser support for a particular version of ember.
+(based on the algorithm described in 
+
+> Whichever browser version is greater/more recent out of:
+
```

Comment:
According to the choose-the-versions algo, 

```
Whichever browser version is greater/more recent out of:

  1. The lowest/least recent version that fulfills any one of these properties:
      - It is the latest version of the browser.
      - It is the latest LTS/extended support version of the browser (such as Firefox ESR).
      - It has at least 0.25% of global marketshare usage across mobile and desktop, based on [statcounter](https://gs.statcounter.com/).
  2. The minimum version supported in the previous release
```


we'd have (if we needed to decide on numbers today, Feb 2nd)

Desktop
- Safari: 15.6+
- Chrome: 103+
- FireFox: 115+ (it seems firefox users are waaay better at staying up to date than Chrome or Safari)

iOS
- Safari 12.2+ (WHY?!)

Android
- Chrome 121+
- FireFox 122+

----


So, while initial motivations are to use the https://github.com/ef4/decorator-transforms plugin instead of the 6 babel plugins all apps need, I don't think we can decide that _at this time_.

But with the evergreen policy, hopefully this changes by the time ember 6.0 comes around.